SSSOM.md

```diff
@@ -491,3 +492,77 @@ Not part of spec, clean up.
 * Can we have more complex logical mappings? A common use case is a taxon-specific anatomy ontology (taxon X). We want to map A to B, but say that B is not equivalent to A, rather it is equivalent to an OWL expression based on A (A and 'in taxon' some X). But there are plenty of arbitrary expressions that could be supported. A while back I looked at some SNOMED to Uberon mappings; it seemed like a lot of the SNOMED terms would better be ('part of' some UBERON:X) rather than directly equivalent to X.
 *  explicitly declare the metadata elements as equivalent to properties?
 * What about mapping to Literals in general? Is this in-spec?
+
+<a name="minimum"></a>
+
+### Minimum Metadata Recommendation
+
+Current mappings are extremely hard to use for data integration, because they are:
+
+- non-transparently imprecise
+- non-transparently incomplete
+- inaccurate
+- unFAIR
+
+In principle, to reach full integration through mappings, you will have to cross-map all ontologies, or semantic
+spaces, which means if you have N "spaces", you have N x N (-1) mappings. To mitigate the explosion
+of mappings, we have to be able to cross-walk:
+- multi-hop forward walks `{ O1:A->O2:A, O2:A->O3:A } --> {O1:A->O3:A}`
+- walk-backs `{ O1:A->O2:A } --> {O2:A->O1:A}`
+- combinations `{ O1:A->O2:A, O2:A->O3:A, O4:A->O3:A } --> {O1:A->O4:A}`
```

Comment:
I think this brings up an important (so far implicit) idea about mappings - are they directed or not? Can we think of a mapping network only as a directed graph (example 1) or as an undirected graph (examples 2 and 3)?

Reply:
I think we must think of them as directed. I believe that in 99.9% of the cases you should be able to do a walk-back if you know what the inverse of the predicate used was; but we should be clear that walk-backs are still "inferences" - and therefore should be treated with slightly less confidence than their corresponding root mapping.